### PR TITLE
Portfolio Website Mobile Version: Add Contact Me Session

### PR DESCRIPTION
**Pull request by rmiyoyo**
# Work Done
- Added the contact me section.
- Added hover effect to the Get-In-Touch button.
- Fixed linter errors.
- Collaborated with my coding partner [Sive](https://github.com/SiveMdluli) on the Contact Me section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -200,5 +202,18 @@ <h2 class="main-text">About Myself</h2>
...
         </div>
     </section>
+    <div class="footer-section">
+        <section class="contact-section">
+            <h1 class="contact-header">Contact me</h1>
+            <p class="contact-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+            <form id="contact-form" action="https://formspree.io/f/xayzpebg" method="POST">
+                <input class="name-box" type="text" name="name" placeholder="Lorem Ipsum" maxlength="30" required>
+                <input class="email-box" type="email" name="email" placeholder="loremipsum@gmail.com" maxlength="500" required>
+                <textarea class="message-box" rows="8" cols="42" name="message" placeholder="Write your message here..." required></textarea>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please from the project requirement, the message field should accept up to 500 chars. So make sure that you give the `maxlength` attribute to the message field. :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @rmiyoyo and @SiveMdluli , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 You have nice design ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rmiyoyo** [comment]
# Work Done After First Review
- Added ```maxlength="500"``` to message field.
- Removed ```maxlength="500"``` from email field.

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**